components/typography/index.css

```diff
@@ -13,6 +13,267 @@ governing permissions and limitations under the License.
+@mixin typographyTShirtSizes .spectrum-DetailM--light, detail-light-M {}
+@mixin typographyTShirtSizes .spectrum-DetailS--light, detail-light-S {}
+
+
+/* Code t-shirt sizes */
+@mixin typographyTShirtSizes .spectrum-CodeXL, code-XL, true {
+  font-family: var(--spectrum-code-xl-text-font-family);
+}
+@mixin typographyTShirtSizes .spectrum-CodeL, code-L, true {
+  font-family: var(--spectrum-code-l-text-font-family);
+}
+@mixin typographyTShirtSizes .spectrum-CodeM, code-M, true {
+  font-family: var(--spectrum-code-m-text-font-family);
+}
+@mixin typographyTShirtSizes .spectrum-CodeS, code-S, true {
+  font-family: var(--spectrum-code-s-text-font-family);
+}
+@mixin typographyTShirtSizes .spectrum-CodeXS, code-XS, true {
+  font-family: var(--spectrum-code-xs-text-font-family);
+}
+
+
+.spectrum-Typography {
+  @mixin typographyTShirtMargins .spectrum-HeadingXXXL {}
+  @mixin typographyTShirtMargins .spectrum-HeadingXXL {}
+  @mixin typographyTShirtMargins .spectrum-HeadingXL {}
+  @mixin typographyTShirtMargins .spectrum-HeadingL {}
+  @mixin typographyTShirtMargins .spectrum-HeadingM {}
+  @mixin typographyTShirtMargins .spectrum-HeadingS {}
+  @mixin typographyTShirtMargins .spectrum-HeadingXS {}
+  @mixin typographyTShirtMargins .spectrum-HeadingXXS {}
+
+  @mixin typographyTShirtMargins .spectrum-BodyXXXL {}
+  @mixin typographyTShirtMargins .spectrum-BodyXXL {}
+  @mixin typographyTShirtMargins .spectrum-BodyXL {}
+  @mixin typographyTShirtMargins .spectrum-BodyL {}
+  @mixin typographyTShirtMargins .spectrum-BodyM {}
+  @mixin typographyTShirtMargins .spectrum-BodyS {}
+  @mixin typographyTShirtMargins .spectrum-BodyXS {}
+}
+
+
+/* T-Shirt sizes for .spectrum-Article */
+.spectrum-Article {
+  font-family: var(--spectrum-body-xl-text-font-family);
+  @mixin typographyTShirtSizes .spectrum-HeadingXXXL, heading-serif-XXXL {}
+  @mixin typographyTShirtSizes .spectrum-HeadingXXL, heading-serif-XXL {}
+  @mixin typographyTShirtSizes .spectrum-HeadingXL, heading-serif-XL {}
+  @mixin typographyTShirtSizes .spectrum-HeadingL, heading-serif-L {}
+  @mixin typographyTShirtSizes .spectrum-HeadingM, heading-serif-M {}
+  @mixin typographyTShirtSizes .spectrum-HeadingS, heading-serif-S {}
+  @mixin typographyTShirtSizes .spectrum-HeadingXS, heading-serif-XS {}
+  @mixin typographyTShirtSizes .spectrum-HeadingXXS, heading-serif-XXS {}
+
+  @mixin typographyTShirtSizes .spectrum-BodyXXXL, body-serif-XXXL {}
+  @mixin typographyTShirtSizes .spectrum-BodyXXL, body-serif-XXL {}
+  @mixin typographyTShirtSizes .spectrum-BodyXL, body-serif-XL {}
+  @mixin typographyTShirtSizes .spectrum-BodyL, body-serif-L {}
+  @mixin typographyTShirtSizes .spectrum-BodyM, body-serif-M {}
+  @mixin typographyTShirtSizes .spectrum-BodyS, body-serif-S {}
+  @mixin typographyTShirtSizes .spectrum-BodyXS, body-serif-XS {}
+  @mixin typographyTShirtSizes .spectrum-BodyXXS, body-serif-XS {}
+
+  @mixin typographyTShirtSizes .spectrum-HeadingXXXL--light, heading-light-XXXL {}
```

Comment:
Not sure why we would include sans-serif classnames under "Article"? I expected to see only the serif options in this wrapper class considering the previously mentioned paradigm of the previous typography

Reply:
whoops, my bad. should be all serif.